includes/admin.form.inc

```diff
@@ -120,11 +120,11 @@ function islandora_internet_archive_bookreader_admin_settings_form(array $form,
       '#type' => 'select',
       '#title' => t('Default page view'),
       '#description' => t('The number of pages to display by default'),
-      '#default_value' => $get_default_value('islandora_internet_archive_bookreader_default_page_view', 1),
+      '#default_value' => $get_default_value('islandora_internet_archive_bookreader_default_page_view', '1up'),
       '#options' => array(
-        1 => 'Single page',
-        2 => 'Side by side',
-        3 => 'Thumbnails',
+        '1up' => 'Single page',
```

Comment:
This should stay as it is. We can do the mapping from 1/2/3 to whatever JS needs in the JS itself on when passing the settings. This change would imply adding an update hook for any server that has already the settings saved.

Reply:
Agreed!